Modules/core/View/Corespaceuser/editAction.twig

```diff
@@ -0,0 +1,195 @@
+                        :class="item.subSection.show ? 'btn btn-primary' : 'btn btn-secondary'"
+                        :value="item.subSection.displayName"
+                        style="margin:5px;"
+                        @click="item.subSection.show = !item.subSection.show">
+                    </input>
+                    <div :id="item.subSection.name + 'Section'" v-show="item.subSection.show">
+                        <div name="tableDiv" v-html="item.subSection.tableHtml"></div>
+                    </div>
+                </div>
+
+                <script v-if="item.module==='booking'" type="module">
+                    import {DynamicForms} from '/externals/pfm/dynamics/dynamicForms.js';
+                    if (!app.$data.dynamicsLoaded) {
+                        let dynamicForms = new DynamicForms();
+                        let spaceId = {{ id_space }};
+                        let sourceId = "resource";
+                        let targets = [
+                            {
+                                elementId: "visa_id",
+                                apiRoute: `revisas/getcategoryvisas/`,
+                                activateOnLoad: true
+                            }
+                        ];
+                        dynamicForms.dynamicFields(sourceId, targets, spaceId);
+                        app.$data.dynamicsLoaded = true;
+                    }
+                </script>
+
+                <script>
+                    function ConfirmDelete(id, name) {
+                        let spaceId = {{ id_space }}
+                        let userId = {{ id_user }}
+                        if (confirm(`Delete ${name} ?`)) {
+                            location.href=`clientsuseraccountsdelete/${spaceId}/${userId}/${id}`;
+                        }
+                    }
+                </script>
+
+            </div>
+        </div>
+
+    </div>
+</div>
+
+<script>
+    class Section {
+        constructor(name, module, btnId, displayName, formHtml = null, tableHtml = null, show = false, subSection = null) {
+            this.name = name;
+            this.module = module;
+            this.btnId = btnId;
+            this.displayName = displayName;
+            this.formHtml = formHtml;
+            this.tableHtml = tableHtml;
+            this.subSection = subSection;
+            this.show = show;
+        }
+    }
+    var app = new Vue({
+        el: '#formSection',
+        data () {
+            return {
+                origin: false,
+                options: [],
+                forms: [],
+                sections: [],
+                params: {
+                    spaceId: {{ id_space }},
+                    userId: {{ id_user }},
+                    lang: 'en'
+                },
+                dynamicsLoaded: false
+            }
+        },
+        mounted: function() {
+            this.origin = JSON.parse(this.$el.attributes['data-origin'].value);
```

Comment:
why not just setting here a the json_encode(...) of the expected data and keep the json directly, why using parameters